src/device/socket/mod.rs

```diff
@@ -7,20 +7,16 @@
 //!
 //! See [`VsockConnectionManager`] for a usage example.
 
-mod error;
 #[cfg(feature = "alloc")]
-mod multiconnectionmanager;
+mod connectionmanager;
+mod error;
 mod protocol;
 #[cfg(feature = "alloc")]
-mod singleconnectionmanager;
```

Comment:
The comment above needs to be updated as well

Reply:
Thanks for spotting that! Fixed.